packages/components/src/internal/models.ts

```diff
@@ -455,7 +455,7 @@ export class EditorModel
                     }
                     row = row.set(col.name, dateVal);
                 } else {
-                    row = row.set(col.name, values.size === 1 ? values.first().raw : undefined);
+                    row = row.set(col.name, values.size === 1 ? values.first().raw?.toString().trim() : undefined);
```

Comment:
would this convert null value to undefined? I think we distinguish between null and undefined for empty value vs delete existing value on update somewhere, but maybe it doesn't matter here.

Reply:
I don't think this does anything different with respect to null vs undefined.